obtainEstimatedWrenches.m

```diff
@@ -106,12 +120,36 @@
 % Print the unknowns to make sure that everything is properly working
 %fullBodyUnknowns.toString(estimator.model())
 
+% Prepare the datastructure that we used in the loop outside the loop 
+% (to improve performances, as otherwise we allocate memory at each 
+% loop step, and allocating memory is usually quite a slow operation) 
 qj_idyn   = iDynTree.JointPosDoubleArray(dofs);
 dqj_idyn  = iDynTree.JointDOFsDoubleArray(dofs);
 ddqj_idyn = iDynTree.JointDOFsDoubleArray(dofs);
 
+    
+% The estimated FT sensor measurements
+estFTmeasurements = iDynTree.SensorsMeasurements(estimator.sensors());
+    
+% The estimated joint torques
+estJointTorques = iDynTree.JointDOFsDoubleArray(dofs);
+    
+% The estimated contact forces
+estContactForces = iDynTree.LinkContactWrenches(estimator.model());
+
+% Sensor wrench buffer 
+estimatedSensorWrench = iDynTree.Wrench();
+
+
 %% For each time instant
-for t=1:size(resampledTime)
+fprintf('obtainEstimatedWrenches: Computing the estimated wrenches\n');
+for t=1:length(resampledTime)
```

Comment:
No, actually I had a problem because size(resampledTime) was returning a vector, something like (actualSize,1) because a vector is a size \times 1 matrix. But I actually don't remember the details, I did that just to get it working and then I assumed they were equivalent.